src/app/signup/member/page.tsx

```diff
@@ -0,0 +1,388 @@
+"use client";
+import { useState } from "react";
+import { Select, Input, Typography, Button, Space, Form, Radio, Flex, Row, Col, Checkbox } from "antd";
+import { useForm, Controller, FormProvider, useFormContext } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { UserFormKey } from "@/constant/common/formKey";
+import { signUpFormSchema } from "@/schema/signup/signUpFormSchema";
+import { district, city, exerciseList, fieldList, type OptionValue } from "@/constant/common/signup";
+
+interface IMemberFormData {
+  [UserFormKey.NICKNAME]: string;
+  [UserFormKey.CELLPHONE]: string;
+  [UserFormKey.SEX]: string;
+  [UserFormKey.AGE]: string;
+  [UserFormKey.CITY]: string;
+  [UserFormKey.DISTRICT]: string;
+  [UserFormKey.GOALS]: string[];
+  [UserFormKey.CATEGORY]: string[];
+}
+
+interface StepProps {
+  next?: () => void;
+  prev?: () => void;
+}
+const { Title, Text } = Typography;
+
+const Step1 = ({ next }: StepProps) => {
+  const {
+    control,
+    formState: { errors, isValid },
+    watch,
+  } = useFormContext<IMemberFormData>();
+  const watchedName = watch(UserFormKey.NICKNAME);
+  const watchedPhone = watch(UserFormKey.CELLPHONE);
+
+  return (
+    <>
+      <Controller
+        name={UserFormKey.NICKNAME}
+        control={control}
+        render={({ field, fieldState }) => (
+          <>
+            <Form.Item
+              style={{ width: "100%" }}
+              validateStatus={errors[UserFormKey.NICKNAME] ? "error" : ""}
+              help={errors[UserFormKey.NICKNAME] ? errors[UserFormKey.NICKNAME]?.message : ""}
+            >
```

Comment:
![스크린샷 2024-09-19 20 47 54](https://github.com/user-attachments/assets/f96fe51e-7f64-468f-9af5-018178c17254)

굳이 상단에 있는 formState에 있는 errors 를 끌어오지 않아도 될 것 같아용
fieldState 내부에 nickname 에 해당하는 에러가 있어서 !

```suggestion
            <Form.Item
              style={{ width: "100%" }}
              validateStatus={errors && "error" }
              help={errors.messages ?? : ""}
            >
```

이렇게 들어오는지 확인은 한 번 해봐야할 것 같은데 이렇게 쓰면 더 깔끔할 것 같아요 : )